Makefile

```diff
@@ -26,6 +26,9 @@ firectl.sha256:
 firectl: $(SRCFILES)
 	go build
 
+docker:
+	docker run --rm -v ${PWD}:/firectl --workdir /firectl golang:1.11 make
```

Comment:
Any reason we shouldn't be using Go 1.12 at this point?

Reply:
No particular reason except that our stated requirement is 1.11.  I can update.